tests/foreman/ui/test_contentview.py

```diff
@@ -4594,6 +4594,114 @@ def test_positive_promote_CV_with_custom_user_role_and_filters(self):
 
     @run_only_on('sat')
     @tier2
+    def test_positive_create_puppet_env_with_module(self):
+        """Check that puppet environment will be created automatically once
+        content view that contains puppet module is published, no matter
+        whether 'Force Puppet' option is enabled or disabled for that content
+        view
+
+        :id: af553367-e621-41e8-86cb-091ba7ba6c0a
+
+        :expectedresults: puppet environment is created and has expected name
+
+        :BZ: 1437110
+
+        :CaseLevel: Integration
+        """
+        repo_url = FAKE_0_PUPPET_REPO
+        puppet_module = 'httpd'
+        self.setup_to_create_cv(
+            repo_url=repo_url, repo_type=REPO_TYPE['puppet'])
+        with Session(self) as session:
+            for force_value in [True, False]:
+                cv_name = gen_string('alpha')
+                # Create content-view
+                make_contentview(
+                    session, org=self.organization.name, name=cv_name)
+                self.content_views.update(
+                    name=cv_name, force_puppet=force_value)
+                self.content_views.add_puppet_module(
+                    cv_name,
+                    puppet_module,
+                    filter_term='Latest',
+                )
+                self.content_views.publish(cv_name)
+                # Form environment name variable for our test
+                env_name = 'KT_{0}_{1}_{2}_{3}'.format(
+                    self.organization.name,
+                    ENVIRONMENT,
+                    cv_name,
+                    str(entities.ContentView(
+                        name=cv_name,
+                        organization=self.organization,
+                    ).search()[0].id),
```

Comment:
Bad indentation. It looks as if `str().search()`, no indentation after opening bracket of `ContentView(`, and, basically, it's against PEP8. How about
```python
                    str(
                        entities.ContentView(
                            name=cv_name,
                            organization=self.organization,
                        ).search()[0].id
                    ),
```